### ISSUE TITLE
GLTFExporter infinity loop while export

### ISSUE THREAD

**Issue by mortalV** [mrdoob/three.js#14216]
##### Description of the problem

I'm using gltfexport sample for my scene and found that process not finished.

I have group in buffergeometry with Infinity count which cause infinity loop in 
function getMinMax( attribute, start, count )

I'm not sure that is normal Infinity count in group but I've always use it and for render all is ok.

##### Three.js version

- [x] r93

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by mrdoob** [comment]
> I have group in buffergeometry with Infinity count which cause infinity loop in
function getMinMax( attribute, start, count )

Wait, what? Why do you have a group with Infinity count?

**Comment by mortalV** [comment]
Many monthes ago when groups was added to library I've found answer on SO like that
https://stackoverflow.com/a/49708915/4640932
In my app I have 2 meshes on the same position:
1. for normal objects
2. for selected objects
And when I need select some object(s) I've exclude from render by groups some part of normal geometry and include some part of selected geometry.
But when no selected objects I must show all geometry from first mesh and I use group from start to Infinity.

BTW similar behaviour used in drawRange https://threejs.org/docs/#api/core/BufferGeometry.drawRange

If Infinity is a problem - OK. I will change my behaviour to real count but it require more operations for calculate actual length
